games_services/lib/src/models/player_data.dart

```diff
@@ -0,0 +1,24 @@
+class PlayerData {
+  // can be null on Android due to privacy settings
```

Comment:
Can you make it documentation comment? 
/// instead of // 
Same for teamPlayerID

Reply:
Done.